requirements.txt

```diff
@@ -2,4 +2,8 @@
 # doc/source/usersguide/installation.rst
 numpy>=1.19.0
 numexpr>=2.6.2
+# blosc2 wheel is actually only needed when compiling on conda envs.
+# Otherwise, lib comes bundled in PyTables wheels (but it doesn't hurt either).
+blosc2>=0.6.1
 packaging
+py-cpuinfo>=9.0.0
```

Comment:
`cpuinfo` should be also added to the `pyproject.toml` if we no longer vendor it.